console/console-init/ui/src/modules/iot-device/containers/AddCredentialsContainer/AddCredentialsContainer.tsx

```diff
@@ -50,6 +50,17 @@ export const AddCredentialsContainer: React.FC<{
     await setCredentialQueryVariable(variable);
   };
 
+  const shouldDisabledSaveBUtton = () => {
```

Comment:
U of Button is capitalized.

Reply:
I'd suggest renaming this to 'isSaveEnabled'. This change should reflect in all similar methods used elsewhere.